From: Tommy Webb <tommy@calyxinstitute.org>
Date: Thu, 15 Feb 2024 18:21:32 +0000
Subject: Temp hardcode content settings types for CalyxOS

This is now handled with a mojom file which apparently does not support
the #include directive. This will ultimately need to be reworked.
For now, hard-code our known content settings types.

Change-Id: Ia77a1ab82358719fb00d5397009c2d24e837e76b
---
 .../core/common/content_settings_types.mojom              | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/components/content_settings/core/common/content_settings_types.mojom b/components/content_settings/core/common/content_settings_types.mojom
--- a/components/content_settings/core/common/content_settings_types.mojom
+++ b/components/content_settings/core/common/content_settings_types.mojom
@@ -388,7 +388,13 @@ enum ContentSettingsType {
   WEB_PRINTING,
 
   NUM_TYPES_CHROMIUM,
-#include "components/content_settings/core/common/bromite_content_settings.inc"
+
+  // Content setting for timezone customization functionality.
+  TIMEZONE_OVERRIDE,
+  VIEWPORT,
+  WEBGL,
+  WEBRTC,
+
   NUM_TYPES_BROMITE,
 };
 
-- 

